ssh/tunnel.py

```diff
@@ -81,6 +82,9 @@ def tunnel(user: str, key_path: str, host: str, port: int=22):
     logger.debug('Closing SSH Tunnel: ' + ' '.join(close_tunnel))
     check_call(close_tunnel)
 
+    if os.path.exists(tunnel_socket.name):
+        os.remove(tunnel_socket.name)
```

Comment:
The `os.remove()` you added at the end won't get called if an exception is raised before it's reached, so this creates possible execution paths where the tempfile doesn't get cleaned up. I'd suggest creating the tempfile within another context manager that's responsible for checking if it's present before attempting to delete it.